tekton/ci-workspace/jobs/tekton-docs-reviews.yaml

```diff
@@ -0,0 +1,149 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: pr-request-reviewer
+  annotations:
+    description: Add reviewers to a GitHub PR
+spec:
+  params:
+    - name: package
+      description: "{owner}/{repo}"
+    - name: pullRequestNumber
+      description: The pull request number
+    - name: github-token-key
+      description: Name of the GitHub token key / file
+    - name: reviewers
+      description: Command separated list of github user slugs
+      default: ""
+    - name: team-reviewers
+      description: Command separated list of github team slugs
+      default: ""
+    - name: github-host
+      default: "github.com"
+  workspaces:
+    - name: github
+  steps:
+  - name: set-reviewers
+    image: python:3.9-alpine3.13
+    script: |
+      #!/usr/bin/env python
+      import json
+      import http.client
+
+      # Load the GitHub Token
+      github-token = ""
+      with open("$(workspaces.github.path/$(params.github-token-key", "r") as gh:
```

Comment:
`$(workspaces.github.path` -> I think you are missing a `)` here?
Similarly with `$(params.github-token-key`.

Reply:
Nice catch, thank you!